code/modules/medical/pathology/pathogen_tools.dm

```diff
@@ -397,7 +397,7 @@
 /obj/item/serum_injector
 	name = "Pathological Injector"
 	desc = "A specialized injector for injecting patients with serums and vaccines."
-	icon = 'icons/obj/pathology.dmi'
+	icon = 'icons/obj/machinery/pathology.dmi'
```

Comment:
Same as blood slide, though *arguably* an injector is a machine by the technical definition rather than how we use it.